test/acceptance/features/rabbit.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Rabbit client
+
+  @rabbit
+  Scenario: Publish and subscribe a text message
+    Given I generate a UUID and store it in context "CorrelationId"
```

Comment:
There is no need to establish a correlation for each scenario. Each scenario creates a random correlation as shown in the code bellow:
https://github.com/Telefonica/golium/blob/36764ff47170fc520c74fa41126b1fefcb00f562/steps/rabbit/session.go#L58

And when the session publish a message if no correlation id is configured, by the use of rabbit standar properties, it is set to the correlator stored in session:
https://github.com/Telefonica/golium/blob/36764ff47170fc520c74fa41126b1fefcb00f562/steps/rabbit/session.go#L184-L188

Also, I do not see why this correlator should be checked in all tests, as it is something provided by the rabbitmq dependency and it should work. IMHO, this should be check once in a scenario where the step to add standard rabbitmq properties is used. @cbrea

Reply:
But @ismtabo, if you do not know what the correlator is then when tests run in parallel you could consume the one you do not want.
i.e.: Scenario 1 and 2 runs in parallel, and both publish a message and consume it in two steps, but one is for users and other for feeds entities.
Step consuming message could consume the message is not expecting, so that, the idea is to add your own correlator to the message in the step to then specifiy that you want to consume the exactly correlated message in order to check correctly and not mixing.
If you let the step to generate internally the correlator you do not know which correlator to wait for in the consumer.
Am I wrong @cbrea?

Reply:
As we'd like to prepare the scenarios for a parallel execution we've to options:

-  Force correlation with CorrelationId ( as we've currently implemented in this PR)
-  Use a different topic in each escenario: "test-rabbit-json-properties-topic"  ( as implemented in Redis feature )

I see a good approach the first one, also we're validating the steps parameterization

Reply:
I understand, but I think that the point is that these ATs should check that the golium steps for  rabbit mq should work as expected. 

Although, the concurrency problems you mentioned are already solved in golium ATs by using different topics for each scenario. So, each session is consuming only its own messages. 

The example you mentioned will be present in the projects that used golium. Its solution will depend in the use of rabbit mq of that project, and it should be implemented by the user of golium.

The correlation generated in each scenario follows the same behavior as in the other steps, such as http, and its use is optional to the user. For rabbit mq, can be overwritten in the publishing by using the property CorrelationId. In addition other standard properties can also be overwritten (e.g.: message id, content type, type, priority, ...).

Reply:
Ok, I agree with you. Let me refactor these ATs to look less redundant